CHANGELOG.md

```diff
@@ -10,9 +10,12 @@ and this project adheres to [Semantic Versioning](http://semver.org/).
 ### Added
 
 - Ability to enable/disable tabs via the reactive `disabled` in tab panes https://github.com/Textualize/textual/pull/3152
+- `MarkdownViewer` now correctly handles anchors in links https://github.com/Textualize/textual/pull/2941
+
 ### Fixed
 
 - Could not hide/show/disable/enable tabs in nested `TabbedContent` https://github.com/Textualize/textual/pull/3150
+- Fixed a crash in `MarkdownViewer` when clicking on a link containing an anchor https://github.com/Textualize/textual/issues/3094
```

Comment:
Just a heads up that when `main` is merged into this branch, these entries could end up in the wrong place.

Reply:
Aye, ta, I tend to try and make a final ChangeLog check before doing a merge, if I get to do the merge.